source/agora/test/Base.d

```diff
@@ -577,6 +578,9 @@ public struct TestConf
 
     /// max failed requests before a node is banned
     size_t max_failed_requests = 100;
+
+    /// The threshold. If not set, it will default to the number of nodes
+    size_t threshold;
```

Comment:
I think we already added something like this in the meantime. A rebase should resolve this.